hivemq-edge/src/frontend/src/api/hooks/useEvents/__handlers__/index.ts

```diff
@@ -49,7 +49,7 @@ export const mockEdgeEvent = (n = maxEvents): Event[] =>
     payload: contentType[x % 3],
 
     source: makeID(sourceKeys[x % sourceKeys.length] as TypeIdentifier.type, x),
-    associatedObject: makeID(sourceKeys[x % sourceKeys.length] as TypeIdentifier.type, x),
+    associatedObject: makeID(sourceKeys[x % sourceKeys.length] as TypeIdentifier.type, x + 100),
```

Comment:
I understand the move, but would argue that for readability purposes, you might benefit from using a appropriately-named `const` rather than a magic number so people understand where the offset comes from more clearly.